samples/04_gis_analysts_data_scientists/visualize_monthly_changes_in_Hirakund_reservoir_using_video.ipynb

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
<p style="user-select: auto;">Add a conclusion saying the sample notebook can be repeated for any other location and for any other time interval by changing the <code style="user-select: auto;">extent</code>  variable and the time interval when calling the <code>create_movie()</code>  function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>